### PR TITLE
SCRUM-13: Add Vehicle DDL,DML,UML

### PR DESCRIPTION
**Pull request by nyotie-tao**

### CODE REVIEW

**Comment by nyotie-tao** [comment]
All queries successfully executed locally.

File: doc/schema/vehicle.puml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+!procedure $unique($name)
+<color:#Red><&key></color> $name
+!endprocedure
+
+!procedure $index($name)
+<color:#Green><&key></color> $name
+!endprocedure
+
+title "OAuth2"
```
Comment (Hannibal0416) [inline doc/schema/vehicle.puml:33]: The title should be "Vehicle".

**Comment by Hannibal0416** [comment]
I think we can add an image field for all the tables; it will store the image and the data format is TEXT. (frontend can use data-uri to render the image.)

**Comment by nyotie-tao** [comment]
Sure I will add it

**Comment by nyotie-tao** [comment]
@Hannibal0416 added

**Review by Hannibal0416 — APPROVED** [review]
Looks good!

**Pull request merged**